### PR TITLE
[TASK-21] As a user, I can navigate through survey questions - Unit tests

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/surveyor/projects/1#card-45376645

## What happened
Added test cases for #28 to make the PRs manageable and easily reviewable.
 
## Insight
Added test cases for question builder files -
image

## Proof Of Work
CI should pass with the test cases.

### CODE REVIEW

File: src/components/WheelSelect/index.js [hunk 1]
```diff
@@ -67,12 +67,13 @@ const WheelSelect = ({ options, multipleSelect }) => {
                 className="wheel-select__container"
                 key={ index }
               >
-                <div className={ elementTextClass }>{ element.display }</div>
+                <div className={ elementTextClass } data-test-id="select-display">{ element.display }</div>
```
Comment (carryall) [inline src/components/WheelSelect/index.js]: How about naming this `wheel-select`?
```suggestion
                <div className={ elementTextClass } data-test-id="wheel-select">{ element.display }</div>
```

File: src/components/WheelSelect/index.js [hunk 2]
```diff
@@ -67,12 +67,13 @@ const WheelSelect = ({ options, multipleSelect }) => {
...
 
                 { multipleSelect && (
                   <div
                     className="wheel-select__checkbox-container"
                     id={ `wheel-select__checkbox--${index}` }
+                    data-test-id="multiple-select"
```
Comment (carryall) [inline src/components/WheelSelect/index.js]: And this as `wheel-multiple-select`
```suggestion
                    data-test-id="wheel-multiple-select"
```

File: src/tests/screens/SurveyDetail/QuestionBuilders/buildQuestionList.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    questions.forEach((question, index) => {
+      const question_details = included.find(
+        (details) => details.id === question.id
+      );
+
+      questions[index]['type'] = question_details.attributes.display_type;
+      questions[index]['pick'] = question_details.attributes.pick;
+      questions[index]['text'] = question_details.attributes.text;
+      questions[index]['answers'] = question_details.relationships.answers.data;
```
Comment (carryall) [inline src/tests/screens/SurveyDetail/QuestionBuilders/buildQuestionList.test.js]:
```suggestion
      question['type'] = question_details.attributes.display_type;
      question['pick'] = question_details.attributes.pick;
      question['text'] = question_details.attributes.text;
      question['answers'] = question_details.relationships.answers.data;
```

File: src/tests/screens/SurveyDetail/QuestionBuilders/determineQuestionType.test.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    it('renders the Rating component', () => {
+      const testId = 'rating-list'
+      const type = 'heart'
+      const pick = 'one'
+
+      const { getByTestId } = render(
+        <DetermineQuestionType type={ type } pick={ pick } />
+      )
+
+      expect(getByTestId(testId)).toBeInTheDocument();
```
Comment (carryall) [inline src/tests/screens/SurveyDetail/QuestionBuilders/determineQuestionType.test.js]: @rafayet-monon Should we also expect the specific emoji to be rendered?

Comment (rafayet-monon) [reply src/tests/screens/SurveyDetail/QuestionBuilders/determineQuestionType.test.js]: @carryall added in [8c6a30b](https://github.com/rafayet-monon/surveyor/pull/29/commits/8c6a30b3eea3defe47eea260928efa8002013686)

**Review by carryall — APPROVED** [review]

**Pull request merged**